### ISSUE TITLE
Changes in AC Freedom application

### ISSUE THREAD

**Issue by makleso6** [liaan/broadlink_ac_mqtt#61]
Hi! There were some changes in recently AC Freedom updated app.
Now it send another UDP packet. I sniff packets via `rvictl` and `tcpdump`.
Now packet looks:
`4500009c59d2000040119c8dc0a8014cc0a801550050da3400885c1b5aa5aa555aa5aa5503000000e4071b3813041109000000005501a8c034da0000a8d60000000007000000000000000000000000002a4ec0a8014c05e79634ea34e5a5a5e5858be696afe7a9bae8b08300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000201`
or
`5aa5aa555aa5aa5503000000e407180114041109000000005501a8c0a8cb0000d4d60000000007000000000000000000000000002a4ec0a8014c05e79634ea34e5a5a5e5858be696afe7a9bae8b08300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000201`

Before application sent
`5aa5aa555aa5aa550000000000000000000000000000000000000000000000002bdc00002a4e6a001cb405e79634ea3401000000c3c30000d8dbaf7f6169b89423ffb28fe8ae6ca2b2f04ae99bba374ed177dd6410f788ae`

 Probably it is breaking changes in future.

**Comment by liaan** [comment]
Thanks
In theory if you don't update aircon firmware, it should stay working.....

Either way, sure one will be able to reverse engineer it again.

**Comment by ThisIsQasim** [comment]
> In theory if you don't update aircon firmware, it should stay working.....

But we can't disable auto update so this will break eventually for everyone

**Comment by ThisIsQasim** [comment]
I can hop on discord with you if you need more info to debug the new firmware

**Comment by ThisIsQasim** [comment]
going by https://github.com/liaan/broadlink_ac_mqtt/issues/76 it looks like there is some auth in place now

**Comment by ThisIsQasim** [comment]
removing the AC from the AC Freedom app has fixed it for now. Guess it needed some freedom from that horrible app

**Comment by aliusc** [comment]
Hello, I have AUX JSMART air conditioner. I can confirm that issue. I have scanned network with same command:
`sudo tcpdump -v | grep 192.168.1.185`
IP address of AC is took from my router - I can see this address in connected devices list. No luck when tried discovery and other ways. Control in AcFreedom app worked well. Then I made **WiFi reset** to AC and started pairing with AC Freedom app. At first I added Wifi information, then app listed all discovered AC and asked to select mine. I executed `monitor.py` and it found my AC! I cancelled pairing in app and not see AC in my app now, but it works in script. If I finish setup by adding AC in app I cannot connect to it using `monitor.py` after script restart. On restart I get error:
```
Starting Monitor...
Authentication Failed to AC
__init__() should return None, not 'bool'
Stopping
```
So, to use this script make your device discoverable, but do not finish setup by adding it to your AcFreedom application on Android.

By the way, when used only app I saw in logs AC is established and holded 3 connections to Amazon servers, but was not discoverable by script.
